admin_ui/src/components/BulkUpdateModal.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+    <Modal v-on:close="$emit('close')">
+        <template v-if="schema">
+            <p>Select a column to update</p>
+
+            <select name="property" v-model="propertyName">
+                <option :value="schema.primary_key_name">
+                    Please select column
+                </option>
+                <option
+                    :key="key"
+                    :value="key"
+                    v-for="(property, key) in schema.properties"
+                >
+                    {{ key }}
+                </option>
+            </select>
+            <p v-show="propertyName != schema.primary_key_name">
+                Value to change
+            </p>
+
+            <form v-on:submit.prevent="updateRows($event)">
+                <div :key="key" v-for="(property, key) in schema.properties">
+                    <InputField
+                        v-if="key == propertyName"
+                        v-bind:format="property.format"
+                        v-bind:isFilter="isFilter"
+                        v-bind:key="property.title"
+                        v-bind:title="property.title"
+                        v-bind:type="property.type || property.anyOf[0].type"
+                        v-bind:choices="property.extra.choices"
+                    />
+                </div>
+                <button>Update</button>
+            </form>
+        </template>
+    </Modal>
+</template>
+
+
+<script lang="ts">
+import InputField from "../components/InputField.vue"
+import Modal from "../components/Modal.vue"
+import * as i from "../interfaces"
+
+export default {
+    props: {
+        schema: Object,
+        tableName: String,
+        selectedRows: {
+            type: Array,
+            defult: () => []
+        },
+        isFilter: {
+            type: Boolean,
+            default: false
+        }
```

Comment:
We can probably drop this prop - I don't think it's needed.

On `InputField`, just set `isFilter` to `false`. 

```vue
<InputField
    v-if="key == propertyName"
    v-bind:format="property.format"
    v-bind:isFilter="false"
    v-bind:key="property.title"
    v-bind:title="property.title"
    v-bind:type="property.type || property.anyOf[0].type"
    v-bind:choices="property.extra.choices"
/>
```

Reply:
You're right. I will change that.